TombEngine/Game/control/box.cpp

```diff
@@ -1049,10 +1026,9 @@ int SearchLOT(LOTInfo* LOT, int depth)
 		node->nextExpansion = NO_BOX;
 	}
 
-	return true;
+	return 1;
```

Comment:
No. Change function declaration to bool instead, if it doesn't return anything but 0 or 1.